protostar-rust/src/main.rs

```diff
@@ -1,30 +1,37 @@
-use anyhow::{Context, Result};
+use anyhow::{anyhow, Context, Result};
 use camino::Utf8PathBuf;
 use clap::Parser;
+use include_dir::{include_dir, Dir};
 use scarb_metadata::MetadataCommand;
+use std::path::PathBuf;
+use tempfile::{tempdir, TempDir};
 
 use rust_test_runner::pretty_printing;
 use rust_test_runner::run_test_runner;
 
+static CORELIB: Dir = include_dir!("../cairo/corelib/src");
```

Comment:
why is this static? It's used only in one place.

Reply:
The directory is being included in the binary at the compile time so there's no reason to do the same with the path to it.

Reply:
Discussed offline

> ... there's no reason to do the same with the path to it.

It should be ("not" missing, therefore confusion)

```
... there's no reason not to do the same with the path to it.
```